### PR TITLE
Add other jsx and styling

### PR DESCRIPTION
**Pull request by anita00001**
## In this branch, we have:
- [ ] Added other sections (specifically, `ProgressDetails` and `CompleteStatus)`
- [ ] Style the webpage using CSS to match following design
image

### CODE REVIEW

File: src/components/BookDetails.jsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ import { useSelector, useDispatch } from 'react-redux';
 import { useEffect } from 'react';
 import { fetchBooks } from '../redux/Books/booksSlice';
 import Buttons from './buttons';
+import CompleteStatus from './CompleteStatus';
+import ProgressDetails from './ProgressDetails';
+import '../styles/BookDetails.css';
```
Comment (zieeco) [inline src/components/BookDetails.jsx:8]: - Hey, please, kindly look up above from line 1 - 6 to remove the commented code to keep your codebase clean.

File: src/components/CompleteStatus.jsx [hunk 2]
```diff
@@ -5,6 +5,32 @@
 // |   |-- ProgressDetails
 // |-- InputBook
```
Comment (zieeco) [inline src/components/CompleteStatus.jsx]: - Please, remove the commented code to keep your codebase clean.

File: src/components/ProgressDetails.jsx [hunk 3]
```diff
@@ -5,6 +5,32 @@
 // |   |-- ProgressDetails
 // |-- InputBook
```
Comment (zieeco) [inline src/components/ProgressDetails.jsx]: - Please, also, remove the commented code to keep your codebase clean.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @anita00001,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookDetails.jsx (hunk 1 above)
Comment (anita00001) [reply src/components/BookDetails.jsx:8]: Thank you so much for your kind review. I have removed all comments.

File: src/components/CompleteStatus.jsx (hunk 2 above)
Comment (anita00001) [reply src/components/CompleteStatus.jsx]: Thank you so much for your kind review. I have removed all comments.

File: src/components/ProgressDetails.jsx (hunk 3 above)
Comment (anita00001) [reply src/components/ProgressDetails.jsx]: Thank you so much for your kind review. I have removed all comments.

**Review by rotimiazeez — APPROVED** [review]
Hi @anita00001 👏🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job cleaning up your codebase as required 👏🏾 
- Good UI ✅️
- App functioning as it should 👌🏽

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**